gnupg/README.md

```diff
@@ -9,3 +9,12 @@ description: |
 ---
 
 Alias for https://webinstall.dev/gpg
```

Comment:
Alias files should _NOT_ have a Files sections because the _real_ file has that section.

Reply:
You're right, my bad, I've dropped the commit.